utils/filesystem/embedfs.go

```diff
@@ -11,7 +10,7 @@ import (
 
 func newEmbedFSAdapter(fs *embed.FS) (afero.Fs, error) {
 	if fs == nil {
-		return nil, fmt.Errorf("%w: missing filesystem", commonerrors.ErrUndefined)
+		return nil, commonerrors.UndefinedVariable("embedded file system", "")
```

Comment:
Would it be better to have two functions so we don't clutter it with the version that has an empty "" string arg